lib/db/database_loader.rb

```diff
@@ -20,12 +20,22 @@ def add_role(filename)
     filename
   end
 
+  # def drop_and_recreate_database
+  #   ENV['DISABLE_DATABASE_ENVIRONMENT_CHECK'] = '1'
+  #   Rake::Task['db:drop'].invoke
+  #   puts "Database dropped"
+  #   Rake::Task['db:create'].invoke
+  #   puts "Database created"
+  # end
+
```

Comment:
Do we still need this commented out version of the method?

Reply:
Let me remove it